test/extended/etcd/etcd_storage_path.go

```diff
@@ -275,7 +275,19 @@ func testEtcd3StoragePath(t g.GinkgoTInterface, kubeConfig *restclient.Config, e
 		gvr("scheduling.k8s.io", "v1alpha1", "priorityclasses"),
 		gvr("settings.k8s.io", "v1alpha1", "podpresets"),
 		gvr("storage.k8s.io", "v1alpha1", "volumeattachments"),
-		gvr("discovery.k8s.io", "v1alpha1", "endpointslices"),
+		gvr("extensions", "v1beta1", "deployments"),
```

Comment:
these look wrong.  Are we enabling endpoints that need to be shut off?  Can we function with them shut off

Reply:
I think that https://github.com/openshift/origin/blob/015a65f45fc0889bb1ac23af430241844a50476a/test/extended/etcd/etcd_storage_path.go#L267

at runtime extensions/v1beta1 are disabled.

Reply:
without it, the test fails
```
fail [github.com/openshift/origin/test/extended/etcd/etcd_test_runner.go:72]: test failed:
etcd data does not match the types we saw:
in etcd data but not seen:
[
	apps/v1beta1, Resource=controllerrevisions 
	extensions/v1beta1, Resource=podsecuritypolicies 
	extensions/v1beta1, Resource=replicasets 
	extensions/v1beta1, Resource=deployments 
	apps/v1beta1, Resource=deployments 
	apps/v1beta2, Resource=replicasets 
	apps/v1beta2, Resource=statefulsets 
	apps/v1beta2, Resource=deployments 
	extensions/v1beta1, Resource=networkpolicies 
	apps/v1beta2, Resource=daemonsets 
	apps/v1beta2, Resource=controllerrevisions 
	apps/v1beta1, Resource=statefulsets 
	extensions/v1beta1, Resource=daemonsets]
seen but not in etcd data:
[]
```

for example https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/24286/pull-ci-openshift-origin-master-e2e-aws-serial/11496

Reply:
`etcd data` is coming from https://github.com/openshift/kubernetes/blob/origin-4.4-kubernetes-1.17.0-rc.2/test/integration/etcd/data.go#L39